source/documentation/runbooks/leavers.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+title: Leavers Guide
+last_reviewed_on: 2021-12-06
+review_in: 3 months
+---
+
+# Leavers Guide
+
+When a team members leave, follow this guide, and log completion in a ticket.
```

Comment:
Probably should be "When a team member leaves"

Reply:
Will update